lib/fetch/index.js

```diff
@@ -88,3 +89,12 @@ export function fetchAllTalks() {
 export function fetchTalk(args) {
   return fetchOne('talk', sanitizeTalk, args);
 }
+
+// TODO: Test? lol
```

Comment:
TODOs all over BB now. I've been adding them any time I find untested code, and there is **a lot** of untested code.